config/packages/_sylius.yaml

```
@@ -22,6 +22,28 @@ sylius_payum:
             - 'sylius'
             - 'stripe_checkout'
 
+sylius_shipping:
+    shipping_method_rules_validation_groups:
+        total_weight_greater_than_or_equal:
```

```suggestion
sylius_shipping:
    shipping_method_rule:
        validation_groups:    
            total_weight_greater_than_or_equal:
```
vs
```suggestion
sylius_shipping:
    shipping_method_rules_validation_groups:
        total_weight_greater_than_or_equal:
```
We need to decide on one, gateway config uses the latter, while the WIP promotions use the former.